### PR TITLE
If 'tests' is disabled, then so should 'external-tests'

### PR DESCRIPTION
**Pull request by levitte**

### CODE REVIEW

File: Configure [hunk 1]
```diff
@@ -665,7 +666,6 @@ while (@argvcopy)
                         delete $disabled{"zlib"};
                         }
 		my $algo = $1;
-		delete $disabled{$algo};
```
Comment (kaduk) [inline Configure]: Why is this line being deleted?

Comment (levitte) [reply Configure]: That's an oops...  I had some experimental code replacing it, but the experiment was cumbersome, so I reverted everything except this line.  Will fix.

**Comment by levitte** [comment]
Disappeared line restored

**Review by kaduk — APPROVED** [review]
Looks good

**Comment by levitte** [comment]
Merged. 302eba3f6dd588f1269aba4605a007ae86b583db

**Comment by dot-asm** [comment]
Shouldn't this be closed? BTW, what does that "rhuijben pushed" mean? I mean why does it show here?

**Comment by levitte** [comment]
Closing.  Re rhuijben, I have *no* idea.  It seems to be a githubbism...  for a while, I was the pusher of all commits, which felt *weird*.

**Pull request closed without merging**